source/server/server.cc

```diff
@@ -322,6 +305,10 @@ void InstanceImpl::initialize(const Options& options,
   // stats.
   stats_store_.setTagProducer(Config::Utility::createTagProducer(bootstrap_));
   stats_store_.setStatsMatcher(Config::Utility::createStatsMatcher(bootstrap_));
+  if (lrs_enabled) {
+    load_reporting_service_store_->setTagProducer(Config::Utility::createTagProducer(bootstrap_));
+    load_reporting_service_store_->setStatsMatcher(Config::Utility::createStatsMatcher(bootstrap_));
```

Comment:
do we need this line? it doesn't make sense to add a stats matcher to this store (or a tag producer) here.